docs/image_captioning.md

```diff
@@ -1,3 +1,6 @@
+---
+layout: default
+---
```

Comment:
the link to lexical constraints needs to be adjusted to point to the Inference subpage:
* `[decoding with lexical constraints](../constraints/README.md).`